core/explorer/core/__main__.py

```diff
@@ -0,0 +1,29 @@
+from explorer.core import model
+
+import click
+
+
+@click.command()
+@click.option(
+    "--out",
```

Comment:
what if I want to output to stdout?

Reply:
Then you're out of luck ;) 

(Added support here: [3faa2d0](https://github.com/tweag/nixpkgs-graph-explorer/pull/56/commits/3faa2d00f3e504a08dc5399a3f894325419a1921))